src/service/alerts/mod.rs

```diff
@@ -64,6 +64,11 @@ pub async fn save(
     alert.stream_name = stream_name.to_string();
     alert.row_template = alert.row_template.trim().to_string();
 
+    // default frequency is 60 seconds
```

Comment:
Can we use env variable to set default frequency

Reply:
added